content/en/blog/2022/announcing-community-manager.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing the OpenTelemetry Community Manager
+linkTitle: Announcing OpenTelemetry Community Manager
+date: 2022-09-24
+author: Austin Parker
+---
+
+OpenTelemetry has demonstrated massive growth since its inception in 2019. What started as a handful of OpenTracing and OpenCensus maintainers and collaborators meeting at the Google campus and over Zoom has now grown into the second-most popular project in the CNCF behind Kubernetes itself. Over 5400 contributors and 700 companies have contributed code, issues, documentation, and invaluable feedback. 
```

Comment:
> What started as a handful of OpenTracing and OpenCensus maintainers and collaborators meeting at the Google campus and over Zoom has now grown into the second-most popular project in the CNCF behind Kubernetes itself.

Put a `,` behind Zoom? Not sure if this works in English, but it helps the reading flow to understand that the injection ends here (I got confused why OTel has grown over zoom...)

Reply:
Done